docs/attributes-registry/cloudfoundry.md

```diff
@@ -0,0 +1,90 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CloudFoundry
+--->
+
+# CloudFoundry
+
+## CloudFoundry Resource Attributes
+
+<!-- semconv registry.cloudfoundry(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `cloudfoundry.app.id` | string | The guid of the application. [1] | `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
+| `cloudfoundry.app.name` | string | The name of the application. [2] | `my-app-name` |
+| `cloudfoundry.instance_id` | string | An index or guid describing the concrete instance of the event source. [3] | `0`; `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
```

Comment:
Does an instance only have an `id` or are there other instance-related properties?
If the latter is the case, I would suggest a different naming: `cloudfoundry.instance.id`

Reply:
This was resolved by splitting the instance id into `app.instance.id` and `system.instance.id`.